### PR TITLE
ci(github): fixed Python pipeline execution

### PR DESCRIPTION
**Pull request by fnk0c**
## :vertical_traffic_light: Quality checklist

- [x] Did you add the changes in the `CHANGELOG.md`?

### CODE REVIEW

File: .github/workflows/python.yaml [hunk 1]
```diff
@@ -9,36 +9,48 @@ jobs:
     name: 'code-check > style:isort'
     runs-on: 'ubuntu-latest'
     container:
-      image: 'ghcr.io/rios0rios0/pipelines/python:3.10-pdm-bullseye'
+      image: 'ghcr.io/fnk0c/pipelines/python:3.10-pdm-bullseye'
```
Comment (rios0rios0) [inline .github/workflows/python.yaml:12]: Could you revert this one? And build the containers to here?

File: .github/workflows/python.yaml [hunk 2]
```diff
@@ -9,36 +9,48 @@ jobs:
...
     steps:
-      - run: pdm run isort --check-only .
+      - uses: actions/checkout@v3
+      - run: |
+          pdm install -v
```
Comment (rios0rios0) [inline .github/workflows/python.yaml:16]: Is that possible to reuse this line in all 3 jobs?

File: github/global/stages/20-security/docker-semgrep/action.yaml [hunk 3]
```diff
@@ -9,7 +9,19 @@ inputs:
 runs:
   using: 'composite'
   steps:
-    - uses: 'actions/checkout@v3'
-    - uses: 'rios0rios0/pipelines/github/golang/abstracts/scripts-repo@main'
-    - run: $SCRIPTS_DIR/global/scripts/semgrep/run.sh "${{ inputs.semgrep_lang }}"
+    - name: 'load'
```
Comment (rios0rios0) [inline github/global/stages/20-security/docker-semgrep/action.yaml:12]: Why do you need those 3 new steps before executing the script?

**Review by rios0rios0 — CHANGES_REQUESTED** [review]
Asking for clarification.

File: .github/workflows/python.yaml (hunk 1 above)
Comment (fnk0c) [reply .github/workflows/python.yaml:12]: I don't have the permission to push to your registry

Comment (rios0rios0) [reply .github/workflows/python.yaml:32]: @fnk0c I don't know how to allow you to do so, but I just built it and pushed. Please check.

File: .github/workflows/python.yaml [hunk 4]
```diff
@@ -4,41 +4,78 @@ on:
...
+    container:
+      image: 'ghcr.io/fnk0c/pipelines/python:3.10-pdm-bullseye'
+    steps:
+      - uses: 'actions/checkout@v3'
+      - name: 'Cache PDM packages'
+        uses: 'actions/cache@v3'
+        with:
+          path: .venv/
+          key: ${{ runner.os }}-pdm-${{ hashFiles('pdm.lock') }}
+          # restore-keys: |
```
Comment (rios0rios0) [inline .github/workflows/python.yaml:19]: Commented code?

File: .github/workflows/python.yaml (hunk 2 above)
Comment (rios0rios0) [reply .github/workflows/python.yaml]: I mean, the same line 3 times. Is that possible to re-use it somehow?

**Review by rios0rios0 — CHANGES_REQUESTED** [review]
Changes requested.

File: .github/workflows/python.yaml (hunk 2 above)
Comment (fnk0c) [reply .github/workflows/python.yaml:16]: Also, by creating a new file having the `pdm install -v` command would still require us to implement the uses. Therefore, we would keep having the same amount of line we have today

**Review by rios0rios0 — APPROVED** [review]

**Pull request merged**